english/count-the-marshmellows/eat-all-the-marshmellows/get-breakfast/go-to-marsh.md

```diff
@@ -0,0 +1 @@
+You race your old Pontiac as fast as it would go to the edge of town, where the marsh begins. A mist hugs its grassy contours. You shout, "Mr. Mallow! Mr. Mallow!" in every direction.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.